src/components/team-gallery/team-gallery.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <section class="grid">
+    <ul class="team-gallery__list">
+      <li
+        v-for="member in team "
+        :key="member.id"
+        class="team-gallery__list-item"
+      >
+        <app-link
+          :to="$localeUrl({ name: 'team-slug', params: { slug: member.slug } })"
+          class="team-gallery-member__link"
+          tabindex="-1"
+        >
+          <dato-image
+            class="team-gallery-member__image"
+            :src="member.image.url"
+            :alt="member.name"
+            :width="member.image.width"
+            :height="member.image.height"
+            loading="lazy"
+            sizes="(min-width: 800px) 20vw, (min-width: 500px) 33vw, 50vw"
+          />
+        </app-link>
+        <div class="team-gallery-member__details">
+          <app-link
+            :to="$localeUrl({ name: 'team-slug', params: { slug: member.slug } })"
+            class="team-gallery-member__link"
+          >
+            <p class="h4">
+              {{ member.name }}
+            </p>
+          </app-link>
+          <span
+            v-if="member.jobTitle"
+            class="body-detail"
+          >{{ member.jobTitle }}</span>
+        </div>
+      </li>
+    </ul>
+  </section>
+</template>
+
+<script setup lang="ts">
+  defineProps<{
+    team: Array<{
+      id: string,
+      slug: string,
+      name: string,
+      lastName: string,
+      jobTitle: string,
+      image: {
+        url: string,
+        width: number,
+        height: number,
+      },
+    }>
+  }>();
+</script>
+
+<style>
+  :root {
+    --team-gallery-grid-columns: 1;
+  }
+
+  .team-gallery__list {
+    display: grid;
+    grid-template-columns: repeat(var(--team-gallery-grid-columns), 1fr);
+    gap: 16px;
```

Comment:
Could you maybe use `var(--spacing-small)`?

Reply:
👍🏻